Crashlytics/Crashlytics/Helpers/FIRCLSDefines.h

```diff
@@ -23,9 +23,8 @@
 // These macros generate a function to force a symbol for the containing .o, to work around an issue
```

Comment:
ooo, you get macros in swift??

Reply:
In Swift there are [Compiler Control Statements](https://docs.swift.org/swift-book/ReferenceManual/Statements.html#ID538) but not macros in the sense that you're likely thinking of. This is just a C header file that ended in the PR due to a change in clang-format. :stuck_out_tongue: